guides/source/getting_started.md

```diff
@@ -242,755 +241,1065 @@ different _actions_. Each action's purpose is to collect information to provide
+used a name like `:article_id` in the routes file, then we would need to use
+`params[:article_id]` here too.
 
-TIP: For more information about migrations, refer to [Active Record Migrations]
-(active_record_migrations.html).
+The `show` action finds a particular article with that ID by using the
+`Article` model's method `find`, and passing `find` that `params[:id]`
+parameter. Once the action has found our article, it needs to then display that
+article's information, which will do by attempting to use a view at
+`app/views/articles/show.html.erb`. Let's create that file now and add this
+content:
 
-At this point, you can use a rails command to run the migration:
+```erb
+<h1><%= @article.title %></h1>
 
-```bash
-$ bin/rails db:migrate
+<%= @article.body %>
 ```
 
-Rails will execute this migration command and tell you it created the Articles
-table.
+Now when we go to <http://localhost:3000/articles/1> we will see the article:
 
-```bash
-==  CreateArticles: migrating ==================================================
--- create_table(:articles)
-   -> 0.0019s
-==  CreateArticles: migrated (0.0020s) =========================================
-```
+![Single Article View](images/getting_started/single_article_view.png)
 
-NOTE. Because you're working in the development environment by default, this
-command will apply to the database defined in the `development` section of your
-`config/database.yml` file. If you would like to execute migrations in another
-environment, for instance in production, you must explicitly pass it when
-invoking the command: `bin/rails db:migrate RAILS_ENV=production`.
+Excellent! We now have our second action working in our controller. But in
+order to navigate to it, we have to manually type in
+<http://localhost:3000/articles/1>. That seems a bit silly. Let's change our
+application a little, so that we can navigate to an article by clicking a link
+from the list of articles.
 
-### Saving Data in the Controller
+To add the link to an article, we need to change
+`app/views/articles/index.html.erb`, which currently looks like this:
 
-Back in `ArticlesController`, we need to change the `create` action
-to use the new `Article` model to save the data in the database.
-Open `app/controllers/articles_controller.rb` and change the `create` action to
-look like this:
+```erb
+<h1>Articles</h1>
 
-```ruby
-def create
-  @article = Article.new(params[:article])
+<ul>
+  <% @articles.each do |article| %>
+    <li><%= article.title %></li>
+  <% end %>
+</ul>
+```
 
-  @article.save
-  redirect_to @article
-end
+This code will render an `li` element for each of the articles, and that
+element contains the title of the article. But we can't click on the title to
+go to an article yet! To make that happen, we need to use an `a` tag:
+
+```erb
+<h1>Articles</h1>
+
+<ul>
+  <% @articles.each do |article| %>
+    <li>
+      <a href='/articles/<%= article.id %>'>
```

Comment:
What do you think about using a `link_to` helper here?

`<%= link_to article.title, article_path(article) %>`

or even

`<%= link_to article.title, article %>`